lib/message.js

```diff
@@ -16,9 +16,9 @@ var $ = bitcore.util.preconditions;
  * constructs a new message to sign and verify.
  *
  * @param {String} message
- * @returns {Message}
```

Comment:
For reference, there isn't complete consistency for this in other classes. Address and PrivateKey both have the return documented, while Transaction and TransactionSignature do not.